README.md

```diff
@@ -18,11 +18,43 @@ jobs:
         anaconda_nightly_upload_token: ${{secrets.UPLOAD_TOKEN}}
 ```
 
-Note that we recommend pinning the action against a specific SHA
+> [!IMPORTANT]
+> Note that we recommend pinning the action against a specific SHA
 (rather than a tag), to guard against the unlikely event of upstream
 being compromised.
 
-## Updating the action
+# Removing old nightly builds
+
+This repository also ships with an action to ease removals of older nightly wheels from a channel.
+
+To use this functionality, add the following snippet to your workflow:
+
+```yml
+jobs:
+  steps:
+    ...
+    - name: Remove old wheels
+      uses: scientific-python/upload-nightly-action/remove-wheels@cantknowhashyet # 0.6.0
+      with:
+        n_latest_uploads: ${{ env.N_LATEST_UPLOADS }}
+        anaconda_nightly_upload_organization: "your-organization"
+        anaconda_nightly_token: ${{secrets.ANACONDA_TOKEN}}
+```
+
+Which will remove all but the `n_latest_uploads` latest uploads from the channel. This is useful
+to avoid hosting outdated development versions, as well as to clean up space.
+
+Note that the ``scientific-python-nightly-wheels`` channel, specifically, already removes
```

Comment:
Should we move this up, so the person doesn't read all of this, make a change to their CI, and only then read the part where it's unnecessary?

Reply:
Addressed in e403c81ce80957498a90530b41b9d19ac6ab16f2.

Reply:
Yes, that's reasonable – addressed in e403c81ce80957498a90530b41b9d19ac6ab16f2.